go.mod

```diff
@@ -3,18 +3,18 @@ module github.com/ONSdigital/dp-dimension-importer
 go 1.15
```

Comment:
NB. go v1.16 is available. This can be updated to 1.16 and the CI build and audit jobs can be updated to 1.16.2

Reply:
Thanks! I will do it in the next PR, as you mentioned.